apps/wallet/src/ui/app/staking/delegation-detail/DelegationDetailCard.tsx

```diff
@@ -96,6 +94,11 @@ export function DelegationDetailCard({
         staked: stakedId,
     }).toString()}`;
 
+    const commission = useMemo(() => {
+        if (!validatorData) return 0;
+        return +validatorData?.fields.commission_rate * 100;
```

Comment:
The null property access here should be removed to indicate that it's actually guaranteed that commission rate will not be null resulting in `NaN` (you guard for this but the null property access makes it look like you don't)